src/styles/index.scss

```diff
@@ -1,3 +1,56 @@
+@import '/src/styles/variables';
+
 body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: ($size-day * 7) + ($size-gap * 6);
+  gap: $size-gap;
+
+  &__day {
+    display: flex;
+    height: $size-day;
+    width: $size-day;
+    box-sizing: border-box;
+    align-items: center;
+    justify-content: center;
+    background-color: #eee;
+    border: 1px solid black;
+    transition: all 500ms;
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+    }
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+      }
+    }
+  }
```

Comment:
It's better to separate styles for different blocks and elements into different files. This way, you can keep your styles organized and it's easier to manage them.